core/src/test/kotlin/org/stellar/anchor/sep6/Sep6ServiceTestData.kt

```diff
@@ -11,16 +11,8 @@ class Sep6ServiceTestData {
                     "authentication_required": true,
                     "min_amount": 1,
                     "max_amount": 10000,
-                    "fields": {
-                        "type": {
-                            "description": "type of deposit to make",
-                            "choices": [
-                                "SEPA",
-                                "SWIFT"
-                            ],
-                            "optional": false
-                        }
-                    }
+                    "funding_methods": ["SEPA", "SWIFT"]
```

Comment:
Does it make sense to support the `type` request parameter when the `/info` response returns `funding_methods`?

Reply:
The plan is to support `funding_method` while deprecating `type`. Same as discussion https://github.com/stellar/java-stellar-anchor-sdk/pull/1581#discussion_r1850640966

Reply:
Removing the fields from the `/info` response is effectively removing support for `type`. If the client already supports `funding_methods` would they still send `type` in the transaction requests?

Reply:
> fields is only deprecated. Anchor can still use it if they prefer.
> We removed fields and type from all existing examples because we want to advocate the use of funding_methods for v3

I understand that we are deprecating `fields` in 3.0 but this PR is removing `fields` from the info response which means clients will need update their code to read `funding_methods` from the info response.

https://github.com/stellar/java-stellar-anchor-sdk/blob/3203bb469260bda710aaffb3704c9ebc28a80b7b/core/src/main/java/org/stellar/anchor/sep6/Sep6Service.java#L544-L551

In this case, I don't see why client would continue sending `type` in their transaction requests. I think if the goal is to deprecate `fields` and `type`, it makes sense to continue returning `fields` in the info response.